feeds/product.php

```diff
@@ -38,6 +38,151 @@
 
 DfTools::validateSecurityToken(Tools::getValue('dfsec_hash'));
 
+/**
+ * Global storage for all active measurements.
+ *   - $__metrics_data[$label] holds ['time' => float, 'memory' => int, 'peak' => int]
+ *   - $__metrics_stack is an indexed array of labels, used to pop the most recent.
+ */
+$__metrics_data  = [];
+$__metrics_stack = [];
```

Comment:
According to PrestaShop standards, camelCase is preferred over snake_case (Even knowing that they sometimes violate their own standards :sweat_smile: )

```suggestion
$__metricsStack = [];
```